experiments/DemoFileWrite/DemoFileWrite.cpp

```diff
@@ -0,0 +1,68 @@
+#include <celero/Celero.h>
+
+CELERO_MAIN
+
+class BaseFixture : public celero::TestFixture
+{
+	public:
+		virtual std::vector<int64_t> getExperimentValues() const
+		{
+			std::vector<int64_t> bufferSizes;
+			bufferSizes.push_back(32);
+			bufferSizes.push_back(64);
+			bufferSizes.push_back(128);
+			bufferSizes.push_back(256);
+			bufferSizes.push_back(512);
+			bufferSizes.push_back(1024);
+			bufferSizes.push_back(2048);
+			bufferSizes.push_back(4096);
+			return bufferSizes;
+		}
+
+		virtual int64_t getExperimentValueSize() const
+		{
+			return 1;
```

Comment:
Perhaps you could make this ```(1024x1024)``` and use it to scale your output, if that's what you were looking to do?  (As part of moving this towards a more general-purpose language.)  If that is the case, perhaps this should even be something like ```virtual double getExperimentValueResultScale() const```?